packages/ckeditor5-dev-release-tools/lib/tasks/updateversions.js

```diff
@@ -59,6 +65,36 @@ module.exports = async function updateVersions( { packagesDirectory, version, cw
 	}
 };
 
+/**
+ * @param {String} cwd
+ * @param {Array.<String>} globPatterns
+ * @param {UpdateVersionsPackagesDirectoryFilter|null} packagesDirectoryFilter
+ * @returns {Promise.<Array.<String>>}
+ */
+async function getPackageJsonPaths( cwd, globPatterns, packagesDirectoryFilter ) {
```

Comment:
I see this function is copied in here:
https://github.com/ckeditor/ckeditor5-dev/pull/920/files#diff-e8b7e66eee579967e36ebc8f45594ff4b8eeacb021f4a8397cb8a02716aaef56R89

Maybe it's a good idea to extract a util?

Reply:
I prefer creating a new util when a thing is repeated thrice and more. So far, I would KISS.